priv/repo/migrations/20170803180536_add_notification_configuration.exs

```diff
@@ -0,0 +1,17 @@
+defmodule CoursePlanner.Repo.Migrations.AddNotificationConfiguration do
+  use Ecto.Migration
+
+  def up do
+    alter table(:users) do
+      add :notified, :date
+      add :notification_frequency_days, :integer, null: false, default: 1
```

Comment:
with having a notification_frequency per user we don't need the system_variable "NOTIFICATION_FREQUENCY" could you delete it from the seed file ?